docs/management/managing-remote-clusters.asciidoc

```diff
@@ -1,26 +1,64 @@
-[[managing-remote-clusters]]
-== Managing Remote Clusters
+[[working-remote-clusters]]
+== Working with remote clusters
+
+{kib} *Management* provides user interfaces for working with data from remote 
+clusters and managing the cross cluster replication process. Replicating indices from a 
+leader remote cluster to a follower index in a local cluster can be used to provide 
+remote backups for disaster recovery or for geo-proximite copies of data.
+
+Before using these features, you should be familiar with the following concepts:
 
-{kib} *Management* provides two user interfaces for working with data from remote 
-clusters. 
+* {xpack-ref}/xpack-ccr.html[Cross cluster replication]
+* {ref}/modules-cross-cluster-search.html[Cross cluster search]
+* {xpack-ref}/cross-cluster-configuring.html[Cross cluster security requirements]
+
+[float]
+[[managing-remote-clusters]]
+== Managing remote clusters 
 
 *Remote Clusters* helps you manage remote clusters for use with 
```

Comment:
The title in the UI is *Add remote clusters*

Reply:
Does the UI change once you have saved clusters? I'm seeing Remote clusters in the UI (although, I should change to sentence case)
<img width="1405" alt="screen shot 2019-02-25 at 2 28 31 pm" src="https://user-images.githubusercontent.com/13408456/53363263-3a2d1500-390a-11e9-98d6-c8074557d12f.png">

Reply:
Oh, I see now.  It's Remote clusters, then you click Add a remote cluster to get to the next screen.  Since the screen you are showing is Add remote cluster.  Just add some wording that first you see the list.  When you add a cluster, you need to define some things like seed node (link to seed node), etc